docker-compose.postgres.yml

```
@@ -0,0 +1,34 @@
+version: "3"
+services:
+  api:
+    container_name: secret-api-with-postgres
+    build:
+      context: ./
+      dockerfile: Dockerfile
+    volumes:
+      - secret:/data/file
+    ports:
+      - "8000:8097"
+    command: [ "server", "--port", "8097", "--path", "/data/file/file.json", "--postgres-url", "postgres:5432" ]
```

Use full connection string for postgres

```suggestion
    command: [ "server", "--port", "8097", "--path", "/data/file/file.json", "--postgres-url", "postgres://postgres:postgres@postgres:5432/postgres" ]
```